### ISSUE TITLE
Focus on navigator handles is lost after it is moved by mouse

### ISSUE THREAD

**Issue by jedrzejruta** [highcharts/highcharts#22205]
#### Expected behaviour
The navigator handle focus is updated even when it's moved by mouse movement.

#### Actual behaviour
The navigator handle focus is lost and is updated only by keyboard events.

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/0zegjLx1/

1. Focus into the left navigator handle using keyboard, then move it using right arrow. 
2. Click on the navigator handle and drag it using mouse.
3. The navigator handle focus isn't updated.

#### Product version
Since introduction of navigator handles in a11y (Highcharts v11.2.0)
